doc/nrf/ug_bt_qualification.rst

```diff
@@ -0,0 +1,109 @@
+
+By using pre-qualified designs, you can omit the testing phase during your product's qualification.
+
+The following product types are used by Nordic Semiconductor:
+
+* End product
+* Controller Subsystem product
+* Host Subsystem product
+* Profile Subsystem product
+* Component product
+
+All versions of Controller Subsystems (:ref:`ug_ble_controller_softdevice` versions), Host Subsystems, and Components are created and qualified by Nordic Semiconductor by default.
+Some Profile Subsystems (for example, :ref:`ug_bt_mesh` versions) are qualified as well.
+If you want to use a Profile Subsystem that is not on the `Bluetooth Qualification Listing`_ yet, run tests through the `Profile Tuning Suite (PTS)`_ tool and qualify the product by yourself.
+
+Qualification process
+*********************
+
+The product qualification cannot be inherited, hence you must complete this process by yourself.
+You can either perform a full qualification process for a new product, or edit an existing entry for a product that has already been qualified.
+
+New entry creation
+==================
+
+The only way to qualify your product is to complete the `Bluetooth SIG's Qualification Process`_ under your member company's account.
+To check if your company is already a SIG member, go to `Join Bluetooth SIG`_ page and search the Member Directory list.
+
+Log into `Launch Studio`_, the official qualification tool, that will guide you through one of the following processes depending on the state of your product:
+
+* `Qualification process with no required testing`_ - If your complete product uses only pre-qualified End products or Subsystem products.
+* `Qualification process with required testing`_ - If your complete product uses a new Bluetooth design or pre-qualified Component products.
+
+Existing entry update
+=====================
+
+Update an existing qualification entry through `Launch Studio`_ in the following cases:
+
+* You significantly changed an existing product's design so that the testing is required.
+* Your product can be added as a new model to one of your existing qualifications.
+* You switched to another SDK version for an existing product.
+
+.. note::
+
+   An existing entry can be edited only by the person that completed the listing.
+   If you want to change the entry owner, contact SIG support.
+
+Switching to another SDK version
+--------------------------------
+
+If you want to switch to another SDK version for an existing product, complete the following steps:
+
+1. Go to the `Nordic Semiconductor Infocenter`_ and search for the SoC or SiP model you use.
+
+#. Navigate to the relevant Compatibility Matrix directory.
+
+#. Open the Bluetooth QDIDs article and, based on the table, confirm the following information:
+
+  a. That relevant Host and SoftDevice Controller Subsystems implemented in the SDK version you want to use are pre-qualified.
+     If yes, perform one of the following actions:
```

Comment:
```suggestion
     If yes, check the following:
```

Reply:
Wouldn't say 'perform the following action' since the first bullet says that no action is required. Also, I feel that the main idea for these 2 is to determine further steps based on this check, so this word might be more fitting